### PR TITLE
Add Kamon metrics for view statistics.

### PR DESCRIPTION
**Pull request by wwajerowicz**

### CODE REVIEW

File: src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/package.scala [hunk 1]
```diff
@@ -31,6 +39,206 @@ package object indexing {
...
+        .add("project", project.show)
+        .add("organization", project.organizationLabel)
+        .add("viewId", view.id.show)
+        .add("sourceId", source.id.show)
+        .add("projectionId", projection.view.id.show)
+        .build()
+
+    def gaugesFor(metricName: String, view: CompositeView, project: Project): Map[String, Gauge] =
+      (view.sources.map { source =>
+        view.progressId(source.id) -> Kamon
```
Comment (umbreak) [inline src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/package.scala]: This has been changed to `source.id.asString`

File: src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/package.scala [hunk 2]
```diff
@@ -31,6 +39,206 @@ package object indexing {
...
+        projection <- view.projections
+        source     <- view.sources
+      } yield view.progressId(source.id, projection.view.id) ->
+        Kamon
+          .gauge(metricName)
+          .withTags(projectionTags(view, project, source, projection)))).toMap
+
+    def countersFor(metricName: String, view: CompositeView, project: Project): Map[String, Counter] =
+      (view.sources.map { source =>
+        view.progressId(source.id) -> Kamon
```
Comment (umbreak) [inline src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/package.scala]: This has been changed to `source.id.asString`

File: src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/package.scala [hunk 3]
```diff
@@ -31,6 +39,206 @@ package object indexing {
...
+        projection <- view.projections
+        source     <- view.sources
+      } yield view.progressId(source.id, projection.view.id) ->
+        Kamon
+          .counter(metricName)
+          .withTags(projectionTags(view, project, source, projection)))).toMap
+
+    def countsFor(view: CompositeView): Map[String, AtomicLong] =
+      (view.sources.map { source =>
+        view.progressId(source.id) -> AtomicLong(0L)
```
Comment (umbreak) [inline src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/package.scala]: This has been changed to `source.id.asString`

**Comment by wwajerowicz** [comment]
> It would be nice to have a Unit Tests for `kamonProjectMetricsFlow`

I don't think it's easy to get the value of a given metric from Kamon. I think it would  be easier to test it through integration tests by scraping the Prometheus reporter endpoint.

**Pull request merged**